storm-server/src/test/java/org/apache/storm/LocalStateTest.java

```diff
@@ -66,6 +66,19 @@ public void testEmptyState() throws IOException {
         Assert.assertNull(ls.get("c"));
         ls.put("a", gs_a);
         Assert.assertEquals(gs_a, ls.get("a"));
+    }
 
+    @Test
+    public void testAllNulState() throws IOException {
+        TmpPath tmp_dir = new TmpPath();
+        String dir = tmp_dir.getPath();
+        LocalState ls = new LocalState(dir, true);
+        GlobalStreamId gs_a = new GlobalStreamId("a", "a");
+        FileOutputStream data = FileUtils.openOutputStream(new File(dir, "12345"));
```

Comment:
Please use try with resources for these streams as well. Fixing the same issues in testEmptyState would be nice too.